src/daemon/daemon.cpp

```diff
@@ -128,13 +130,25 @@ bool t_daemon::run(bool interactive)
       rpc_commands->start_handling(std::bind(&daemonize::t_daemon::stop_p2p, this));
     }
 
+    cryptonote::rpc::DaemonHandler rpc_daemon_handler(mp_internals->core.get(), mp_internals->p2p.get());
+    cryptonote::rpc::ZmqServer zmq_server(rpc_daemon_handler);
+
+    zmq_server.addTCPSocket("*", "31337");
+
+    LOG_PRINT("Starting ZMQ server...", LOG_LEVEL_0);
```

Comment:
I most likely copied a print line from elsewhere in the file, parsed it in my head as "yea, that should do what I want", and went with it.  Can change if desireable.